pages/news/2020-06-07-mixxx-2-3-beta-released.html

```diff
@@ -0,0 +1,88 @@
+title: Mixxx 2.3 beta released
+author: Be.
+date: 2020-06-07 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+We are proud to announce the release of Mixxx 2.3 beta, [download it!]({% url '/download.html' %}) This release brings hotcue colors & labels, Serato metadata import, Rekordbox metadata import, intro & outro cues, a revamped LateNight skin, multithreaded analysis, deck cloning, and more.
+
+<img src="{% static '/static/images/news/screenshot-2.3-beta-deck.png' %}" width="633" height="275" alt="Screenshot of a Mixxx deck with the intro and outro cues and hotcue editing menu"/>
```

Comment:
```suggestion
<img src="{% static '/static/images/news/screenshot-2.3-beta-deck.png' %}" width="633" height="275" alt="Screenshot of a Mixxx deck with the intro and outro cues and hotcue editing menu"/>
```

Can you use a Markdown image and just set max-height? 
Like I suggested here: https://github.com/mixxxdj/website/pull/93#issuecomment-636094693

Right now, the image looks bad on my phone, because the height is fixed:
![Screenshot_20200607_194707_com duckduckgo mobile android](https://user-images.githubusercontent.com/1834516/83976109-4028e880-a8f8-11ea-8eff-05c76e7f832e.jpg)

Reply:
Then the right side gets cut off:
![Screenshot from 2020-06-07 12-59-11](https://user-images.githubusercontent.com/9455094/83976293-bdd00280-a8be-11ea-8f2d-2502fcc6fee8.png)

Reply:
Can you try this patch? I cannot reproduce that the image is cut off neither on my laptop (when shrinking the window) nor on my phone. Maybe you accidently zoomed in?

```udiff
diff --git a/pages/news/2020-06-07-mixxx-2-3-beta-released.html b/pages/news/2020-06-07-mixxx-2-3-beta-released.html
index e1d73e9..2d9b5e4 100644
--- a/pages/news/2020-06-07-mixxx-2-3-beta-released.html
+++ b/pages/news/2020-06-07-mixxx-2-3-beta-released.html
@@ -7,10 +7,10 @@ date: 2020-06-07 01:00:00
 {% block post %}

 {% load markup %}
-{% filter markdown %}
+{% filter markdown:"extra" %}
 We are proud to announce the release of Mixxx 2.3 beta, [download it!]({% url '/download.html' %}) This release brings hotcue colors & labels, Serato metadata import, Rekordbox metadata import, intro & outro cues, a revamped LateNight skin, multithreaded analysis, deck cloning, and more.

-<img src="{% static '/static/images/news/screenshot-2.3-beta-deck.png' %}" width="633" height="275" alt="Screenshot of a Mixxx deck with the intro and outro cues and hotcue editing menu"/>
+![Screenshot of a Mixxx deck with the intro and outro cues and hotcue editing menu]({% static '/static/images/news/screenshot-2.3-beta-deck.png' %}){: style="max-size: 275px" }

 Please test it, [report bugs](https://bugs.launchpad.net/mixxx/+filebug), help [translate](https://mixxx.org/wiki/doku.php/internationalization), and [contribute controller mappings](https://mixxx.org/wiki/doku.php/contributing_mappings). If you want help make Mixxx even more awesome, introduce yourself on our [Zulip chat](https://mixxx.zulipchat.com/#narrow/stream/109123-introduce-yourself) and read the [developer documentation](https://mixxx.org/wiki/doku.php#developer_documentation).

```

Reply:
![Screenshot_20200607_223316_com duckduckgo mobile android](https://user-images.githubusercontent.com/1834516/83979475-f4356e00-a90e-11ea-878e-81077ea87150.jpg)

Reply:
I tested with Chromium and could not reproduce the edge getting cut off. Upon looking closer, I see that other parts of the page are cut off in that screenshot I posted above including the close window button and navbar hamburger menu button.

Now ready for merge?